src/psyclone/psyir/nodes/extract_node.py

```diff
@@ -117,25 +109,45 @@ def dag_name(self):
         '''
         return "extract_" + str(self.position)
 
+    @property
+    def input_list(self):
+        ''':returns: the list of input variables that will be written.
+        :rtype: list of str
+        '''
+        return self._input_list
+
+    @property
+    def output_list(self):
+        ''':returns: the list of output variables that will be written.
```

Comment:
"...the list of variables that are outputs of this extraction region." perhaps?

Reply:
Again that's better.